WISE-Community/WISE-Client
# build(Sass): Upgrade gulp-sass to 5 and replace node-sass with dart-sass

## Test 
- Run ```npm i``` to update to the new devDependencies
- Run ```gulp watch-sass```, change a VLE scss file and see that css was outputted and the change is reflected in the browser

Closes #419
